modules/apps/notification/notification-rest-api/src/main/java/com/liferay/notification/rest/dto/v1_0/util/NotificationUtil.java

```diff
@@ -99,6 +100,13 @@ public static NotificationTemplate toNotificationTemplate(
 					0L);
 		}
 
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-168220")) &&
+			(notificationTemplateId != 0)) {
```

Comment:
@marcelabc why do we need this condition `notificationTemplateId != 0`?

Reply:
I don't know, but I removed this part. Thanks!